e2e-tests/commands/shared-commands/assertions/assert-copy-with-currency-name.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * assertCopyWithCurrencyName
+ * Assert a selector with a currency name and optional question mark
+ * @param {String} pageTitle: Page title
+ * @param {String} currencyName: Currency name
+ * @param {String} currencyName: Currency name
+ * @param {Function} selector: Cypress selector
+ * @param {Boolean} withQuestionMark: Add a question mark to the expected text.
+ */
+const assertCopyWithCurrencyName = ({
+  pageTitle, currencyName, selector, withQuestionMark = false,
+}) => {
+  let expected = `${pageTitle} ${currencyName}`;
```

Comment:
`expectedCopy` would be better than `pageTitle` so it can be used for non page titles

Reply:
Can address this after merge